client/guidestours/index.js

```diff
@@ -67,37 +169,59 @@ class GuidesStep extends Component {
 	}
 
 	addTargetListener() {
-		const { target = false, onNext, type } = this.props;
-		if ( type === 'bullseye' && onNext && target.addEventListener ) {
+		const { target = false, onNext } = this.props;
+		if ( onNext && target.addEventListener ) {
 			target.addEventListener( 'click', onNext );
 		}
 	}
 
 	removeTargetListener() {
-		const { target = false, onNext, type } = this.props;
-		if ( type === 'bullseye' && onNext && target.removeEventListener ) {
+		const { target = false, onNext } = this.props;
+		if ( onNext && target.removeEventListener ) {
 			target.removeEventListener( 'click', onNext );
 		}
 	}
 
+	getBullseyePosition() {
+		const { placement = 'center', target } = this.props;
+		const rect = target
+			? target.getBoundingClientRect()
+			: global.window.document.body.getBoundingClientRect();
+
+		return bullseyePositioners[ placement ]( rect );
+	}
+
 	render() {
-		const { text, type, style, onNext, onQuit } = this.props;
+		const stepPos = this.getStepPosition();
+		const bullseyePos = this.getBullseyePosition();
+		const stepCoords = posToCss( stepPos );
+		const pointerCoords = posToCss( bullseyePos );
+
+		const { text, onQuit } = this.props;
+
 		return (
-			<Card className="guidestours__step" style={ style } >
+			<Card className="guidestours__step" style={ stepCoords } >
 				<p>{ text }</p>
-				{ type !== 'bullseye' &&
-					<Button onClick={ onNext } primary>Next</Button>
-				}
-				<Button onClick={ onQuit } secondary>Skip all</Button>
+				<div className="guidestours__step__extra">
+					<hr />
+					<p>
+						Click the <Gridicon icon={ this.props.icon } size={ 24 } /> to continue …
```

Comment:
We should `i18n` these now, lest we forget. https://github.com/Automattic/wp-calypso/blob/master/client/lib/mixins/i18n/localize/README.md should be perfect for this.